ckanext/switzerland/plugin.py

```diff
@@ -39,14 +40,21 @@
 )
 
 
-class OgdchPlugin(plugins.SingletonPlugin):
+class OgdchPlugin(plugins.SingletonPlugin, DefaultTranslation):
     plugins.implements(plugins.IConfigurer)
     plugins.implements(plugins.IValidators)
     plugins.implements(plugins.IFacets)
     plugins.implements(plugins.IActions)
     plugins.implements(plugins.ITemplateHelpers)
+    plugins.implements(plugins.ITranslation)
 
-    # IConfigurer
+    # ITranslation
+
+    def i18n_locales(self):
+        return get_langs()
+
+    def i18n_domain(self):
+        return 'ckanext-switzerland'
```

Comment:
Since we are using the DefaultTranslation it it provides reasonable defaults, you can remove these methods